### DISCUSSION TITLE
Certificates problem

### DISCUSSION THREAD

**Discussion by miroslavskela** [postalserver/postal#1611] [Help with using Postal]
Hello, great job with postal!
I have problem with enabling tls. I have generated certs using letsencrypt, and added them to .yml file but it doesn't work.
FIles exist on specified location.
If someone could help. Thank you very much.
![Screenshot from 2021-09-29 15-11-35](https://user-images.githubusercontent.com/36046629/135275358-340c308d-8337-4758-8996-06ca120dd484.png)
![Screenshot from 2021-09-29 15-10-06](https://user-images.githubusercontent.com/36046629/135275366-357e7451-7548-4de5-a54c-742053a76368.png)

**Comment by miroslavskela** [comment]
Found answer few posts below.

**Comment by willpower232** [comment]
The files need to be in the config folder otherwise they will not be present in the docker containers.
